<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StationItemSettings</class>
 <widget class="QWidget" name="StationItemSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QWidget" name="SISContainer" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="l_station_caption">
          <property name="text">
           <string>Station settings</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="l_station_name">
          <property name="text">
           <string>&amp;Name:</string>
          </property>
          <property name="buddy">
           <cstring>m_station_name</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_station_name">
          <property name="maxLength">
           <number>64</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="l_station_type">
          <property name="text">
           <string>&amp;Type:</string>
          </property>
          <property name="buddy">
           <cstring>m_station_type</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_station_type"/>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="l_station_cid">
          <property name="text">
           <string>&amp;CID:</string>
          </property>
          <property name="buddy">
           <cstring>m_station_cid</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_station_cid">
          <property name="maxLength">
           <number>16</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_20">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="l_station_SN">
          <property name="text">
           <string>&amp;Serial number:</string>
          </property>
          <property name="buddy">
           <cstring>m_station_ip</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_station_SN">
          <property name="maxLength">
           <number>12</number>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="l_station_ip">
          <property name="text">
           <string>&amp;IP:</string>
          </property>
          <property name="buddy">
           <cstring>m_station_ip</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_station_ip">
          <property name="maxLength">
           <number>16</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="l_station_mask">
          <property name="text">
           <string>N&amp;etmask:</string>
          </property>
          <property name="buddy">
           <cstring>m_station_mask</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_station_mask">
          <property name="maxLength">
           <number>16</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="l_station_gateway">
          <property name="text">
           <string>&amp;Gateway:</string>
          </property>
          <property name="buddy">
           <cstring>m_station_gateway</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_station_gateway"/>
        </item>
       </layout>
      </item>
      <item row="8" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_17">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="l_key_caption">
          <property name="text">
           <string>Key settings</string>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <property name="indent">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="9" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="l_key_id">
          <property name="text">
           <string>&amp;Key:</string>
          </property>
          <property name="buddy">
           <cstring>m_key_id</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_key_id"/>
        </item>
       </layout>
      </item>
      <item row="10" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="l_key_name">
          <property name="text">
           <string>N&amp;ame:</string>
          </property>
          <property name="buddy">
           <cstring>m_key_name</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_key_name">
          <property name="maxLength">
           <number>64</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="11" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="l_key_func">
          <property name="text">
           <string>&amp;Function:</string>
          </property>
          <property name="buddy">
           <cstring>m_key_func</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_key_func"/>
        </item>
       </layout>
      </item>
      <item row="12" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="l_key_dest">
          <property name="text">
           <string>&amp;Destination:</string>
          </property>
          <property name="buddy">
           <cstring>m_key_dest</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_key_dest"/>
        </item>
       </layout>
      </item>
      <item row="13" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="l_key_priority">
          <property name="text">
           <string>&amp;Priority:</string>
          </property>
          <property name="buddy">
           <cstring>m_key_priority</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_key_priority">
          <property name="maxLength">
           <number>16</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="14" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_19">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="l_additional_caption">
          <property name="text">
           <string>Additional settings</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="15" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="l_key_activate_lines">
          <property name="text">
           <string>Acti&amp;vate control lines:</string>
          </property>
          <property name="buddy">
           <cstring>m_key_activate_lines</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_key_activate_lines">
          <property name="maxLength">
           <number>128</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="b_key_activate_lines">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Add active control line</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="16" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_18">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="c_pretone">
          <property name="contextMenuPolicy">
           <enum>Qt::ActionsContextMenu</enum>
          </property>
          <property name="text">
           <string>Pre-tone control</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="17" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="l_key_pretone_play">
          <property name="text">
           <string>Nu&amp;mber:</string>
          </property>
          <property name="buddy">
           <cstring>m_key_pretone_play</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_key_pretone_play">
          <property name="maxLength">
           <number>16</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="18" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="l_key_pretone_duration">
          <property name="text">
           <string>D&amp;uration:</string>
          </property>
          <property name="buddy">
           <cstring>m_key_pretone_duration</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_key_pretone_duration">
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="currentText">
           <string>ONCE</string>
          </property>
          <item>
           <property name="text">
            <string>ONCE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>INFINITE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>times</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>seconds</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="19" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="l_key_pretone_times">
          <property name="text">
           <string>Seconds:</string>
          </property>
          <property name="buddy">
           <cstring>m_key_pretone_play</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_key_pretone_times">
          <property name="maxLength">
           <number>16</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>dataChanged(QString)</slot>
 </slots>
</ui>
